go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/nerdswords/yet-another-cloudwatch-exporter
 go 1.18
 
 require (
-	github.com/aws/aws-sdk-go v1.44.96
+	github.com/aws/aws-sdk-go v1.44.108
```

Comment:
Is this needed for any particular feature? Otherwise I'd leave dependecy upgrade out of the PR.

Reply:
It's a minor version upgrade but not a mandatory requirement for it